src/webViews/tools/dataMdbMigrate.webview.ts

```diff
@@ -0,0 +1,554 @@
+             .separator {
+               flex: 1;
+               border: none;
+               border-top: 1px solid var(--vscode-settings-sashBorder);
+             }
+            
+            #selectedFolder {
+                flex-grow: 1;
+                margin-left: 10px;
+                padding: 10px;
+                border: 1px solid #ccc;
+                border-radius: 5px;
+                background-color: #f9f9f9;
+            }
+
+            .validation-error {
+                color: #ff0000;
+                font-size: 14px;
+                padding: 5px;
+            }
+
+            .redButton{
+                background: #ea2328;
+            }
+            .redButton:hover {
+                background: #bb1117;
+            }
+            
+            .advanced-header {
+                cursor: pointer;
+                font-weight: bold;
+                padding: 5px;
+                display: flex;
+                align-items: center;
+            }
+            
+            .advanced-settings {
+                display: none;
+                padding: 10px;
+            }
+            .advanced-settings.active {
+                display: block;
+            }
+            
+            .arrow-icon {
+                transition: transform 0.2s;
+                margin-right:5px;
+            }
+
+            .tooltip {
+                position: relative;
+                display: inline-block;
+            }
+
+            .tooltip .tooltiptext {
+                visibility: hidden;
+                width: 400px;
+                background-color: #555;
+                color: #fff;
+                text-align: center;
+                border-radius: 6px;
+                padding: 5px 10px;
+                position: absolute;
+                z-index: 1;
+                bottom: 100%; 
+                left: 50%; 
+                margin-left: -40px; 
+                opacity: 0;
+                transition: opacity 0s;
+            }
+
+            .tooltip:hover .tooltiptext {
+                visibility: visible;
+                opacity: 1;
+            }
+
+            .checkbox-container {
+                display: flex;
+                align-items: center; 
+            }
+
+            .checkbox-container label {
+                margin-left: 8px;
+                position: relative;
+                bottom: -2px; 
+            }
+
+            input[type="checkbox"] {
+                width: 16px; 
+                height: 16px; 
+                vertical-align: middle;
+            }
+
+
+            
+          </style>
+        </head>
+
+        <body>
+        <h2 class="heading">Mongodb to Couchbase Data Migration</h2>
+        <form action="#" method="post" id="dataMigrateForm">
+        <p>This tool utilizes the CLI in the background to migrate the data, Please refer to <a href="https://github.com/couchbaselabs/cbmigrate" target="_blank">this</a> for more info on how to use the CLI.</p>
```

Comment:
make it in multiple lines, this is increasing the code's width.

Reply:
yes, added a new line.